idea_town/frontend/static-src/locales/en-US.l20n

```diff
@@ -0,0 +1,68 @@
+<siteName "Idea Town">
+
+<headerSubtitle "The innovative community where new Ideas for Firefox are tested.">
+
+<footerLinkContribute "Contribute">
+<footerLinkCookies "Cookies">
+<footerLinkPrivacy "Privacy">
+<footerLinkLegal "Legal">
+
+<home "Home">
+
+<menuTour "Tour Idea Town">
+<menuWiki "Idea Town Wiki">
+<menuFileIssue "File an Issue">
+<menuLogout "Logout">
+
+<landingWelcome "Welcome to Idea Town">
+<landingIntro "Test new features.<br> Give us feedback.<br> Help build Firefox.">
+<landingFxaGetStartedButton "Get started with your Firefox Account">
+<landingFxaAlternateButton "Don't have an account? Sign up.">
+<landingCtaLegal "By proceeding, you agree to the <a href=\"https://www.mozilla.org/about/legal/terms/services/\">Terms of Service</a> and <a href=\"https://www.mozilla.org/privacy/firefox-cloud/\">Privacy Notice</a> of Idea Town.">
```

Comment:
Per stas' comment about DOM overlays, you can pull the `href`s out of this string